src/app/store/chains/stx-chain.slice.ts

```diff
@@ -7,12 +7,14 @@ import { keySlice } from '../software-keys/software-key.slice';
 interface StxChainKeyState {
   highestAccountIndex: number;
   currentAccountIndex: number;
+  currentAccountStacksDescriptor: string;
 }
 
 const initialState: Record<string, StxChainKeyState> = {
   [defaultWalletKeyId]: {
     highestAccountIndex: 0,
     currentAccountIndex: 0,
+    currentAccountStacksDescriptor: '',
```

Comment:
Ideally it'd be better if the store structure were to save the stacks descriptor for all accounts, then reference it to the effect of `store.keychains.stacks[currentAccountIndex]`.